uaclient/util.py

```diff
@@ -759,6 +759,7 @@ def is_config_value_true(config: Dict[str, Any], path_to_value: str):
     r"(\'token\': \')[^\']+",
     r"(\'X-aws-ec2-metadata-token\': \')[^\']+",
     r"(.*\[PUT\] response.*api/token,.*data: ).*",
+    r"(https://bearer:)[\w-]*",
```

Comment:
Let's make this as flexible as possible to avoid accidentally missing a redact.
```suggestion
    r"(https://bearer:)[^\@]+",
```